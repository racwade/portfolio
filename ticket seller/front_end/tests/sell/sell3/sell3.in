login
test_sell
sell
TestEvent

logout