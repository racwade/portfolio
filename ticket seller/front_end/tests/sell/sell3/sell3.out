welcome to the ticket seller system
enter username:
logged in as test_sell
enter event name:
enter ticket price:
error: must enter a valid ticket price
bye